data_structures/src/serializers.rs

```diff
@@ -0,0 +1,572 @@
+    let peers_command = FlatBuffersPeers::create(builder, &PeersArgs { peers: addresses });
+
+    let message = FlatBuffersMessage::create(
+        builder,
+        &MessageArgs {
+            magic: peers_args.magic,
+            command_type: FlatBuffersCommand::Peers,
+            command: Some(peers_command.as_union_value()),
+        },
+    );
+    build_flatbuffer(builder, message)
+}
+
+// Create a witnet's peers message to decode a flatbuffers' peers message
+fn create_peers_message(peers_args: PeersWitnetArgs) -> Option<Message> {
+    peers_args.peers.peers().map(|ftb_addresses| {
+        // TODO: Refactor as declarative code [24-10-2018]
+        let len = ftb_addresses.len();
+        let mut counter = 0;
+        let mut ftb_address: Option<Address>;
+        let mut peer;
+        let mut vec_addresses = Vec::new();
+        while counter < len {
+            peer = ftb_addresses.get(counter);
+            ftb_address = create_address(peer);
+            if ftb_address.is_some() {
+                vec_addresses.push(ftb_address.unwrap());
+            }
+            counter += 1;
+        }
+        Message {
+            kind: Command::Peers {
+                peers: vec_addresses,
+            },
+            magic: peers_args.magic,
+        }
+    })
+}
+
+// Create a ping flatbuffer to encode a witnet's ping message
+fn create_ping_flatbuffer(
+    builder: &mut FlatBufferBuilder,
+    ping_args: PingFlatbufferArgs,
+) -> Vec<u8> {
+    let ping_command = FlatBuffersPing::create(
+        builder,
+        &PingArgs {
+            nonce: ping_args.nonce.to_owned(),
+        },
+    );
+    let message = FlatBuffersMessage::create(
+        builder,
+        &MessageArgs {
+            magic: ping_args.magic,
+            command_type: FlatBuffersCommand::Ping,
+            command: Some(ping_command.as_union_value()),
+        },
+    );
+
+    build_flatbuffer(builder, message)
+}
+
+// Create a witnet's ping message to decode a flatbuffers' ping message
+fn create_ping_message(ping_args: &PingWitnetArgs) -> Message {
+    Message {
+        kind: Command::Ping {
+            nonce: ping_args.nonce,
+        },
+        magic: ping_args.magic,
+    }
+}
+
+// Create a pong flatbuffer to encode a witnet's pong message
+fn create_pong_flatbuffer(
+    builder: &mut FlatBufferBuilder,
+    pong_args: &PongFlatbufferArgs,
+) -> Vec<u8> {
+    let pong_command = FlatBuffersPong::create(
+        builder,
+        &PongArgs {
+            nonce: pong_args.nonce.to_owned(),
+        },
+    );
+    let message = FlatBuffersMessage::create(
+        builder,
+        &MessageArgs {
+            magic: pong_args.magic,
+            command_type: FlatBuffersCommand::Pong,
+            command: Some(pong_command.as_union_value()),
+        },
+    );
+
+    build_flatbuffer(builder, message)
+}
+
+// Create a witnet's pong message to decode a flatbuffers' pong message
+fn create_pong_message(pong_args: &PongWitnetArgs) -> Message {
```

Comment:
Same thing as with the `create_ping_message`:
```suggestion
fn create_pong_message(pong_args: PongWitnetArgs) -> Message {
```